### PR TITLE
chore: targeting sept updates for Windows 2019 VHD

### PR DESCRIPTION
**Pull request by marosset**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: vhd/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -188,7 +188,7 @@ function Install-WindowsPatches {
 
     # KB4558998 contains August 11, 2020 cumulative updates for Windows Server 2019
     # https://www.catalog.update.microsoft.com/Search.aspx?q=KB4565349
```
Comment (mainred) [inline vhd/packer/configure-windows-vhd.ps1:190]: Maybe we may just change this comment into a template instead for each specific patch.

**Review by mainred — DISMISSED** [review]
The patch URL and image version look good to me. But we need to change the patch comment, approve this PR as I think you'll fix it.

File: vhd/packer/configure-windows-vhd.ps1 (hunk 1 above)
Comment (marosset) [reply vhd/packer/configure-windows-vhd.ps1:190]: Will update

Comment (marosset) [reply vhd/packer/configure-windows-vhd.ps1]: Updated comment, thanks!

**Pull request merged**